### PR TITLE
feat: 5.34.0 upgrade script for PageBuilder

### PR DESCRIPTION
**Pull request by leopuleo**
## Changes
With this PR:
- PageBuilder: add `5.34.0` upgrade page.
- PageBuilder: add `5.34.0` upgrade script -> fetch existing pages and create a folder `link` to ROOT. The script works for all tenants and locales.
- PageBuilder: add `link` to ROOT folder for initialPages during Page Builder install.

Closes [WEB-1952](https://webiny.atlassian.net/browse/WEB-1952)

### CODE REVIEW

File: packages/api-page-builder/src/graphql/crud/system.crud.ts [hunk 1]
```diff
@@ -204,6 +205,13 @@ export const createSystemCrud = (params: CreateSystemCrudParams): SystemCrud =>
                         notFound: notFoundPage.pid
                     }
                 });
+
+                // Create folder link for initialPages
+                await Promise.all(
+                    initialPagesData.map(page =>
+                        context.folders.createLink({ id: page.id, folderId: "ROOT" })
+                    )
+                );
```
Comment (Pavel910) [inline packages/api-page-builder/src/graphql/crud/system.crud.ts]: I would move this to a plugin, like `context.pageBuilder.onSystemAfterInstall` and do this there, just to keep the logic isolated into a separate file which is easy to unplug if necessary.

File: packages/api-page-builder/src/graphql/upgrades/v5.34.0/index.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    const { tenant, pageBuilder, folders, i18n } = params;
+
+    /**
+     * Find all locales for the current tenant, so we can find all pages for each locale.
+     */
+    const [locales] = await i18n.locales.storageOperations.list({
+        where: {
+            tenant: tenant.id
+        },
+        limit: 100
```
Comment (Pavel910) [inline packages/api-page-builder/src/graphql/upgrades/v5.34.0/index.ts:87]: Why limit? If there's no reason, remove this.

File: packages/app-page-builder/src/admin/plugins/upgrades/v5.34.0.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
+// @ts-nocheck
```
Comment (Pavel910) [inline packages/app-page-builder/src/admin/plugins/upgrades/v5.34.0.tsx]: Try removing this, and see what TS complains about.

**Review by Pavel910 — CHANGES_REQUESTED** [review]

File: packages/api-page-builder/src/graphql/upgrades/v5.34.0/index.ts (hunk 2 above)
Comment (leopuleo) [reply packages/api-page-builder/src/graphql/upgrades/v5.34.0/index.ts:87]: `limit` is mandatory for this operation

File: packages/api-page-builder/src/graphql/subscriptions/afterSystemInstall.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { ContextPlugin } from "@webiny/api";
+import WebinyError from "@webiny/error";
+import { PbContext } from "~/types";
+
+export const afterSystemInstall = () => {
+    return new ContextPlugin<PbContext>(async context => {
+        try {
+            context.pageBuilder.onSystemAfterInstall.subscribe(async () => {
```
Comment (leopuleo) [inline packages/api-page-builder/src/graphql/subscriptions/afterSystemInstall.ts:8]: Move try/catch in here

**Review by Pavel910 — APPROVED** [review]

**Pull request merged**